client/my-sites/checkout/checkout/web-payment-box.jsx

```diff
@@ -28,523 +26,383 @@ import {
-							.show()
-							.then( paymentResponse => {
-								analytics.tracks.recordEvent( 'calypso_checkout_apple_pay_submit_payment_sheet', {
-									is_renewal,
-								} );
-
-								const { payerName, details } = paymentResponse;
-								const { token } = details;
-
-								if ( 'EC_v1' !== token.paymentData.version ) {
-									return; // Not supported yet.
-								}
-
-								const cardRawDetails = {
-									tokenized_payment_data: token.paymentData,
-									name: payerName,
-									country: getTaxCountryCode( this.props.cart ),
-									'postal-code': getTaxPostalCode( this.props.cart ),
-									card_brand: token.paymentMethod.network,
-									card_display_name: token.paymentMethod.displayName,
-								};
-
-								setPayment( newCardPayment( cardRawDetails ) );
-								this.props.onSubmit( event );
-								paymentResponse.complete( 'success' );
-							} )
-							.catch( error => {
-								debug( 'Error while showing the payment request', error );
-							} );
-					} catch ( error ) {
-						debug( 'Error while running the payment request', error );
-					}
-				}
-				break;
-
-			case WEB_PAYMENT_BASIC_CARD_METHOD:
-				{
-					const paymentRequest = this.getPaymentRequestForBasicCard();
-
-					try {
-						paymentRequest
-							.show()
-							.then( paymentResponse => {
-								const { details } = paymentResponse;
-								const { billingAddress } = details;
-
-								const cardRawDetails = {
-									number: details.cardNumber,
-									cvv: details.cardSecurityCode,
-									'expiration-date': details.expiryMonth + '/' + details.expiryYear.substr( 2 ),
-									name: details.cardholderName,
-									country: billingAddress.country,
-									'postal-code': billingAddress.postalCode,
-								};
-
-								setPayment( newCardPayment( cardRawDetails ) );
-								this.props.onSubmit( event );
-								paymentResponse.complete();
-							} )
-							.catch( error => {
-								debug( 'Error while showing the payment request', error );
-							} );
-					} catch ( error ) {
-						debug( 'Error while running the payment request', error );
-					}
-				}
-				break;
-
-			default:
-				debug( `Unknown or unhandled payment method ${ paymentMethod }.` );
-		}
-	};
-
-	/**
-	 * @param {string} errorMessage  Error message.
-	 * @param {DOMEvent} event       Button event.
-	 */
-	displaySubmissionError = ( errorMessage, event ) => {
-		event.preventDefault();
-		notices.error( errorMessage );
+		analytics.tracks.recordEvent( 'calypso_checkout_apple_pay_open_payment_sheet', {
+			is_renewal: isRenewal,
+		} );
+		paymentRequest.show();
 	};
+	if ( paymentType === 'apple-pay' ) {
+		return <button className="web-payment-box__apple-pay-button" onClick={ onClick } />;
+	}
+	return (
+		<button
+			className="web-payment-box__web-pay-button button checkout__pay-button-button button is-primary button-pay pay-button__button"
+			onClick={ onClick }
+		>
+			{ translate( 'Select a payment card' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Select a payment card', { context: 'Loading state on /checkout'} )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).